server/base/src/main/java/org/apache/accumulo/server/util/ZooZap.java

```diff
@@ -61,6 +61,10 @@ private static void message(String msg, Opts opts) {
     boolean zapTservers = false;
     @Parameter(names = "-tracers", description = "remove tracer locks")
     boolean zapTracers = false;
+    @Parameter(names = "-coordinators", description = "remove compaction coordinator locks")
```

Comment:
What about calling it `coach` instead of `coordinator`. A Compaction Coach or compaction-coaches. It is a new term, its a lot shorter and has a similar meaning. We could even shorten them to `comp-coaches` or `co-coaches`.

Reply:
> I'm not sure where the confusion would come to play.

I have a whole set of thoughts on naming that answers why I think it's more confusing, but I can't get it down to a reasonable size for a response here. I can try, if it's important. If it's not important, then maybe I can just discuss offline sometime.

In short, I think `-compaction-coordinators` is better here. Yes, it's long, but it's completely unambiguous, and matches with the name of the `KeywordExecutable` service and `bin/accumulo` argument to launch it.

Reply:
@ctubbsii - I didn't take your original comment to mean that you wanted the parameter name `-coordinators` changed to `-compaction-coordinators`. I'll make this change in the next commit.